tests/phpunit/Configuration/ConfigurationFactoryTest.php

```diff
@@ -66,6 +74,19 @@ public static function tearDownAfterClass(): void
         self::$mutators = null;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
is it needed for test's `setUp()`?

Reply:
it's not needed, but I put it by habit as an inherited method. I would like us to address this convention though; opening up a ticket for it

Reply:
#860